app/main.py

```diff
@@ -1 +1,53 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
```

Comment:
You are using `append()` method to add a single value to a list. According to the checklist, you should use `extend()` method or `+=` operator to add multiple values to a single list. If you want to add a single value, `append()` is fine, but for multiple values, use `extend()` or `+=`.